tools/windows-node-installer/pkg/cloudprovider/aws/ec2.go

```diff
@@ -301,6 +312,15 @@ func (a *awsProvider) authorizeSgIngress(sgID string, vpc *ec2.Vpc) error {
 					(&ec2.IpRange{}).
 						SetCidrIp(myIP + "/32"),
 				}),
+			(&ec2.IpPermission{}).
+				SetIpProtocol("tcp").
+				// winrm port
+				SetFromPort(5986).
```

Comment:
Please move the port number to a constant, and specify that is the WinRM https port. WinRM also has an http port (5985), and we dont want to confuse them.

Reply:
While you're changing things here, please move the rdp port above to a constant as well.

Reply:
I suggest doing that in a separate commit. Ideally it should be a separate PR but I can let that go.

Reply:
You have not made the winrm ansible https port into a constant. It is making the RDP port becoming a constant that requires a separate PR.

Reply:
I will added that to the future PR that comes after this one.

Reply:
Why? You are introducing the port in this PR and it is a two line change to make it a constant. Please don't add to tech debit if possible.

Reply:
Got it. Added.